test/index.js

```diff
@@ -162,6 +162,33 @@ describe('gulp-awspublish', function () {
       stream.end();
     });
 
+    it('should not send s3 header x-amz-acl if option {noAcl: true}', function (done) {
+
+      var stream = publisher.publish({}, {noAcl: true});
+      stream.write(new gutil.File({
+        path: '/test/hello.txt',
+        base: '/',
+        contents: new Buffer('hello world')
+      }));
+
+      stream
+        .pipe(es.writeArray(function (err, files) {
+          expect(err).not.to.exist;
+          expect(files).to.have.length(1);
+          expect(files[0].s3.path).to.eq('test/hello.txt');
```

Comment:
no but you are writing to s3, and I think we have some publisher.sync tests that read what is on s3